core/src/main/scala/kafka/admin/LogDirsCommand.scala

```diff
@@ -83,7 +84,10 @@ object LogDirsCommand {
     }
 
     private def createAdminClient(opts: LogDirsCommandOptions): JAdminClient = {
-        val props = new Properties()
+        val props = if (opts.options.has(opts.commandConfigOpt))
+            Utils.loadProps(opts.options.valueOf(opts.commandConfigOpt))
+        else
+            new Properties()
         props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, opts.options.valueOf(opts.bootstrapServerOpt))
         props.put(AdminClientConfig.CLIENT_ID_CONFIG, "log-dirs-tool")
```

Comment:
Can we use props.putIfAbsent() for client_id config, so that users can overwrite client_id in props file.

Reply:
Thanks for the suggestion. I will make the change as suggested.